### ISSUE TITLE
Remove alpha tag from Navigation component

### ISSUE THREAD

**Issue by tochwill** [BBC-archive/psammead#2558]
**Is your feature request related to a problem? Please describe.**
The new version of the Navigation component will be an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**
Remove alpha tag after successful a11y swarm on the component.

**Describe alternatives you've considered**

**Testing notes**

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by tochwill** [comment]
Blocked on https://github.com/bbc/psammead/issues/2559
